generated/nidcpower/session.py

```diff
@@ -705,19 +767,22 @@ class _SessionBase(object):
         var = session['0,1'].power_line_frequency
     '''
     power_source = attributes.AttributeEnum(attributes.AttributeViInt32, enums.PowerSource, 1150000)
-    '''
+    '''Data Type: PowerSource
+
     Specifies the power source to use. NI-DCPower switches the power source used by the  device to the specified value.
     Default Value: NIDCPOWER_VAL_AUTOMATIC
     is set to NIDCPOWER_VAL_AUTOMATIC. However, if the session is in the Committed or Uncommitted state  when you set this attribute, the power source selection only occurs after you call the  niDCPower_Initiate function.
 
     Note: Automatic selection is not persistent and occurs only at the time this attribute
     '''
     power_source_in_use = attributes.AttributeEnum(attributes.AttributeViInt32, enums.PowerSourceInUse, 1150001)
-    '''
+    '''Data Type: PowerSourceInUse
```

Comment:
I'm not sure about this, but for enums, would it make sense to type them as `nidcpower.PowerSourceInUse`? This is how a user would type it by default.

Reply:
Fixed